src/app/page.tsx

```diff
@@ -1,3 +1,4 @@
+import Banner from '@/components/common/Banner/Banner';
```

Comment:
move Banner component from common to /features/home/sections/BannerSection/index.tsx